packages/CodingStyle/src/Application/UseImportsAdder.php

```diff
@@ -77,7 +77,22 @@ public function addImportsToNamespace(
         );
 
         $newUses = $this->createUses($useImportTypes, $functionUseImportTypes, $namespaceName);
-        $namespace->stmts = array_merge($newUses, $namespace->stmts);
+        $wasAdded = false;
+
+        foreach ($namespace->stmts as $key => $stmt) {
+            if (! $stmt instanceof Use_) {
+                continue;
+            }
+
+            // add use statements now
+            array_splice($namespace->stmts, $key + 1, 0, $newUses);
```

Comment:
I think 2nd and 3rd argument (`$key + 1, 0`) have to be somehow modified, to make the @Route edge case work.